tests/unit/tf/examples/test_usecase_transformers_next_item_prediction.py

```diff
@@ -36,12 +38,9 @@ def generate_date():
         df.to_csv('{tmpdir}/train_set.csv')
         """
     )
-    tb.cells[4].source = tb.cells[4].source.replace("get_booking('/workspace/data')", "")
-    tb.cells[4].source = tb.cells[4].source.replace(
-        "read_csv('/workspace/data/train_set.csv'", f"read_csv('{tmpdir}/train_set.csv'"
-    )
-    tb.cells[31].source = tb.cells[31].source.replace("epochs=5", "epochs=1")
-    tb.cells[37].source = tb.cells[37].source.replace("/workspace/ensemble", f"{tmpdir}/ensemble")
+    tb.cells.pop(6)
+    tb.cells[29].source = tb.cells[29].source.replace("epochs=5", "epochs=1")
+    tb.cells[36].source = tb.cells[36].source.replace("/workspace/ensemble", f"{tmpdir}/ensemble")
```

Comment:
we should not be replacing lines... these lines should be dictated by an environment variable. So that the logic is set with a given start location and everything else builds on that. Should come from the INPUT and OUTPUT env vars that would be read at the beginning of the notebook.

Reply:
change made